src/swagger-theme-dark.scss

```diff
@@ -0,0 +1,18 @@
+
+@use 'swagger-dark' as sw;
+
+[data-themes*='dark'] {
+	@include sw.main-swagger;
+
+	@include sw.def-swagger;
+}
+
+@media (prefers-color-scheme: dark)
+{
+	@include sw.main-swagger;
+
+	[data-themes='default'] {
```

Comment:
Should this include also be in the selector?
Otherwise it will apply whenever the users OS theme is dark regardless of the theme that is selected in Nextcloud.

Reply:
In fact, include in Sass is not an include, as in other languages. It includes definitions that you'll use later in Selector.
Then you can see the: all the "dark" definitions are within @media or data-themes selectors.

Reply:
But why is this include here and not in the selector like in the case above?

Reply:
Sorry, you're right. Now it produces the same CSS.